### DISCUSSION TITLE
Is it possible to have rounded angles ?

### DISCUSSION THREAD

**Discussion by RobinLbt** [pierpo/react-archer#198] [General]
The line style is still "angle", but with rounded angles

Is it possible to achieve?

**Comment by pierpo** [comment]
Hey!

I looked it up: I was hoping that the smooth property of the svg path would be enough (like stroke-linejoin), but it's not.

So, having more smooth angles would imply more drawing logic: we'd need to add smooth-points in the paths, which is not trivial unfortunately 😅 
It's doable though, you can still fork the lib and include this logic.

I'm suggesting the fork because right now the architecture of the lib does not allow for easy customization of the paths without complexifying the whole codebase.

I hope it helps!

Here are the smooth points rough idea (very very rough)

<img width="1309" alt="image" src="https://github.com/pierpo/react-archer/assets/6313316/6baaa2cb-dfc2-409a-8a43-a052fa16ce16">

**Reply by RobinLbt** [reply]
Will try to do something, merci !
